specificatie/openapi.yaml

```diff
@@ -648,6 +810,115 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoek opdracht voor het zoeken naar resource binnen een geometrie.'
+      type: object
+      required:
+        - geometrie
+        - spatialOperator
+      properties:      
+        geometrie:
+          description: 'De geometrie waarbinnen gezocht moet worden naar objecten. Alleen Point en Polygon worden ondersteund. Als Point wordt opgegeven, dan moet ook de paramter radius worden opgegeven zodat in een gebied met een straal rond het opgegeven punt wordt gezocht.' 
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.yaml'
+        spatialOperator:
+          type: string
+          description: >-
+            "Een uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1. Ondersteund worden:
+            - intersects  : object geometrie `overlapt` of `raakt` de opgegeven contour
+            - within      : object geometrie valt volledig binnen de opgegeven contour
+            - contains    : object geometrie omsluit de opgegeven contour volledig"
+          enum:
+            - intersects
+            - within
+            - contains
+          default: intersects
+        radius:
+          description: 'Als geometrie type point wordt meegegeven, dan kan in een straal rondom dat punt worden gezocht door hier de straal op te geven (eenheid meters).'
+          type: number
+          default: 100
+          minimum: 1
+          maximum: 280
+        tolerantie:
+          description: 'Een percentage ter indicatie van het gewenste niveau van precisie voor spatial data (in meters).'
+          type: number
+          default: 1
+          minimum: 1
+          maximum: 100
+
+    Waardebereik:
+      description: >-
+        Een generieke aanduiding voor de min en max waarde van een attribuut van een object.
+        Als alleen een min waarde wordt opgegeven, dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan de min waarde.
+        Als alleen een max waarde wordt opgegeven, dan wordt er gezocht naar waarden die kleiner dan of gelijk zijn aan de max waarde.
+        Als zowel min en max waarde worden opgegeven waarbij min < max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die tussen min en max liggen.
+        Als zowel min en max waarde worden opgegeven waarbij min > max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die niet tussen min en max liggen. 
+      type: object
+      properties:
+        min:
+          type: number
+        max:
+          type: number
+
+    LijstOperator:
+      type: string
+      enum:
```

Comment:
wij gebruiken overal kleine letters (snake_case) voor enumeratiewaarden. dus ik zou het consistenter vinden om "or", "not", enz. te gebruiken

Reply:
OK